setup/aio/011_Install_Puppet_Server.rb

```diff
@@ -0,0 +1,29 @@
+extend BeakerPuppet::Install::Puppet5
+extend Beaker::DSL::InstallUtils::FOSSUtils
+
+test_name "Install Puppet Server" do
+  server_version = ENV['SERVER_VERSION'] || 'latest'
+  release_stream = ENV['RELEASE_STREAM'] || 'puppet'
+  nightly_builds_url = ENV['NIGHTLY_BUILDS_URL'] || 'http://nightlies.puppetlabs.com'
```

Comment:
This will work, but I think we want to prefer nightlies.puppet.com rather than puppetlabs.